CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+Version 12.0
```

Comment:
Similar to qiskit-on-iqm I don't think in this case we break any existing functionality.

Reply:
Yes, I suppose these can be minor updates instead.